app/views/posts/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1>Create a New Post</h1>
+
+<% flash.each do |name, msg| -%>
+  <%= content_tag :div, msg, class: name %>
+<% end -%>
+<%= form_with model: [:user, :post, @comment], data: { turbo: false }, local: true do |f| %>
```

Comment:
- [ ] When I visit the new post page, it provides an error. Kindly fix it.

  ![image](https://github.com/SaveryIV/blog-app-on-Rails/assets/54848363/b31110c9-132b-4f52-9206-6d05588c7539)

- [ ] After fixing the issue with the post creation, kindly ensure the comments and likes can be created as well, and work without errors.

Reply:
@DammyShittu The problem is doesn't exist yet any user so gives that error

Reply:
I solve it